lib/ddg-atb.js

```diff
@@ -37,13 +37,18 @@ function redirect(e) {
 }
 
 function update(){
-    var atb     = ss.storage.atb, 
-        setATB  = ss.storage.set_atb;
+    var atb = ss.storage.atb; 
 
-    if(!atb || !setATB){
+    if(!atb){
         return;
     }
 
+    if(!ss.storage.atb_set){
+        ss.storage.atb_set = true;
+    }
```

Comment:
I'm still getting mixed up here. If a user has `atb` and  no `set_atb` then we want to return or do we continue to get them a valid `set_atb`?